Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -32,7 +33,7 @@ public function __construct(SymfonyFixturesLoader $fixturesLoader)
         $this->fixturesLoader = $fixturesLoader;
     }
 
-    protected function configure()
+    protected function configure() : void
```

Comment:
This is a BC break: the class is not final and an extending class may override this method.